komodo/data/__init__.py

```diff
@@ -0,0 +1,17 @@
+import os
+
+
+class Data(object):
+    def __init__(self, extra_data_dirs=None):
+        """Create a Data object responsible for giving you the filez you need.
+        Takes a list of extra_data_dirs to search."""
+        self._dirs = [os.path.dirname(__file__)]
+        if extra_data_dirs is not None:
+            self._dirs.extend(extra_data_dirs)
```

Comment:
should we validate that the extra dirs are existing?

Reply:
done